ryubing-ryujinx: init at 1.2.86

yehorb opened the pull request:
Ryujinx project have moved to their own GitLab platform for development.

The current Ryujinx package links to incorrect packages, so updates do not work.

There are already PRs to fix the issue, linked below.

My version is differs in checkver, it uses GitLab instance API to get the latest version. And I also add Canary release package.

Closes [#1337](https://github.com/Calinou/scoop-games/issues/1337)
Relates to [#1346](https://github.com/Calinou/scoop-games/pull/1346) [#1339](https://github.com/Calinou/scoop-games/pull/1339)

- [x] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

Calinou commented:
Looks good to me, but I wonder if we really have to rename `ryujinx` to `ryubing-ryujinx`. The official name of this continuation is still Ryujinx on its website, so I think that's what users will try to install instinctively.

Could you rename the manifests to `ryujinx` and `ryujinx-canary`? This should be good to merge afterwards.

yehorb commented:
@Calinou, I rebased and squashed the changes into two commits.

I am not sure why the checker is upset.

It states that:

```text
String 'https://git.ryujinx.app/api/v4/projects/ryubing%2Fryujinx/repository/tags?search=^Canary-' does not validate against format 'uri'.
```

But it is a valid URL that opens in a browser without any issues.

It may be upset about the `^` character, but that how tag search is done in the [GitLab API](https://docs.gitlab.com/api/tags/).

yehorb commented:
I did not know that, thank you for pointing it out. GilLab docs were kind enough to point out how to encode a / character, but did not mention anything about ^ character.

Calinou reviewed (approved):
Thanks!
